.gitignore

```
@@ -20,3 +20,7 @@ src/test/data/sandbox/
 # MacOS custom attributes files created by Finder
 .DS_Store
 docs/_site/
+
+# Binary files
+.class
+/bin/
```

```suggestion
/bin/

```
Add newline at end of file to pass Java CI